apiary.apib

```diff
@@ -83,24 +83,139 @@ TODO
 
 
 # Group bootstrap
+Show or edit the list of bootstrap peers.
+
+Running 'ipfs bootstrap' with no arguments will run 'ipfs bootstrap list'.
+
+SECURITY WARNING:
+
+The bootstrap command manipulates the "bootstrap list", which contains
+the addresses of bootstrap nodes. These are the *trusted peers* from
+which to learn about other peers in the network. Only edit this list
+if you understand the risks of adding or removing nodes from this list.
 
 ## add [GET /bootstrap/add]
+Add peers to the bootstrap list
+
+Outputs a list of peers that were added (that weren't already
+in the bootstrap list).
+
++ Parameters
+    + arg (Multiaddr, required) - A peer to add to the bootstrap list
+    + default (boolean, optional) - add default bootstrap nodes
```

Comment:
You need add these to the query above, like this `[GET /bootstrap/add{?arg,&default}]` See https://github.com/apiaryio/api-blueprint/blob/master/API%20Blueprint%20Specification.md#examples for details